qwp_encoder_test.go

```
@@ -0,0 +1,2080 @@
+	// schema + no column data (0 rows).
+	off := qwpHeaderSize + 2 // +2 for empty delta symbol dictionary
+	nameLen, n, _ := qwpReadVarint(msg[off:])
+	off += n
+	off += int(nameLen) // skip name
+	rowCount, n, _ := qwpReadVarint(msg[off:])
+	off += n
+	if rowCount != 0 {
+		t.Fatalf("rowCount = %d, want 0", rowCount)
+	}
+	colCount, n, _ := qwpReadVarint(msg[off:])
+	off += n
```

golangci-lint (2.11.4)

[error] ineffectual assignment to off

(ineffassign)